pkgs/dart_tooling_mcp_server/lib/src/mixins/dtd.dart

```diff
@@ -168,6 +232,25 @@ base mixin DartToolingDaemonSupport on ToolsSupport {
     inputSchema: InputSchema(),
   );
 
+  static final _hotReloadTool = Tool(
+    name: 'hot_reload',
```

Comment:
Note that this name ends up being user visible, so we actually probably want to make these more human readable (and spaces are fine)